birdhouse/components/cowbird/config/cowbird/config.yml.template

```diff
@@ -153,8 +153,7 @@ sync_permissions:
             type: route
           - name: "{outputID}"
             type: route
-      # see 'optional-components/secure-data-proxy' for more details on protected WPS-outputs
-      wps_outputs:
```

Comment:
Lost the key "wps_oputputs", expected?

Reply:
Yep, the `wps_outputs` key here and `weaver` key above are supposed to be Magpie service types, but they are not actual service types. Instead, all their contained values now use the key `api` which is a valid Magpie service type.

Reply:
In fact, without this change, all cowbird hooks fail with:

```
cowbird.config.ConfigErrorInvalidServiceKey: Service `weaver` used in sync config is not valid since it was not found in Magpie services (['access', 'api', 'geoserver', 'geoserverwfs', 'geoserverwms', 'geoserverwps', 'ncwms', 'thredds', 'wfs', 'wps']).
```

so I think this change is necessary to ensure that cowbird works properly with this default config file.

If we're still waiting on https://github.com/Ouranosinc/cowbird/pull/35 to merge in this PR can we at least get the bug fix merged in the meantime. I've factored out that portion of this PR here: #336

Reply:
The cowbird PR is still a work in progress, so I was still waiting to merge this PR.

We can add the necessary part from your PR for now, to avoid the error, and I will add the rest when the other PR is done.